README.md

```diff
@@ -1,167 +1,87 @@
 
-Recommender systems require massive datasets to train, particularly for deep learning based solutions. The transformation of these datasets in order to prepare them for model training is particularly challenging. Often the time taken to complete the necessary steps, such as feature engineering, categorical encoding, and normalization of continuous variables exceeds the time it takes to train a model.
+NVTabular is a component of [**NVIDIA Merlin Open Beta**](https://developer.nvidia.com/nvidia-merlin). NVIDIA Merlin is used for building large-scale deep learning recommender systems, which require massive datasets to train, particularly for deep learning based solutions. The transformation of these datasets in order to prepare them for model training can be challenging. Often the time taken to complete the necessary steps, such as feature engineering, categorical encoding, and normalization of continuous variables, exceeds the time it takes to train a model. With NVTabular being a part of the Merlin ecosystem, it also works with the other Merlin components including [HugeCTR](https://github.com/NVIDIA/HugeCTR) and [Triton Inference Server](https://github.com/NVIDIA/tensorrt-inference-server) to provide end-to-end acceleration of recommender systems on the GPU. Extending beyond model training, with NVIDIA’s Triton Inference Server, the feature engineering and preprocessing steps performed on the data during training can be automatically applied to incoming data during inference.
 
-With NVTabular being a part of the Merlin ecosystem, it also works with the other Merlin components including [HugeCTR](https://github.com/NVIDIA/HugeCTR) and [Triton Inference Server](https://github.com/NVIDIA/tensorrt-inference-server) to provide end-to-end acceleration of recommender systems on the GPU.
+## Benefits 
 
-NVTabular is designed to support data scientists and machine learning (ML) engineers train (deep learning) recommender systems and resolve tabular data problems by allowing them to:
+Our ultimate goal is faster iteration on massive tabular datasets, both for experimentation during training, and also for production model responsiveness. NVTabular is designed to support data scientists and machine learning (ML) engineers train (deep learning) recommender systems and resolve tabular data problems by allowing them to:
 
-* prepare datasets quickly and easily for experimentation so that more models can be trained
+* prepare datasets quickly and easily for experimentation so that more models can be trained.
 * work with datasets that exceed GPU and CPU memory without having to worry about scale.
 * focus on what to do with the data, and not how to do it, using our abstraction at the operation level.
```

Comment:
I think this benefits section should include some extra info:

*  I think one of the core benefits we should highlight is the accelerated dataloaders - we should add a section about quickly training models with Tensorflow, PyTorch.  
* We should mention inference + integration with Triton